frontend/app/routes/$lang+/_protected+/alerts+/index.tsx

```diff
@@ -0,0 +1,24 @@
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { redirect } from '@remix-run/node';
+
+import { getSubscriptionService } from '~/services/subscription-service.server';
+import { featureEnabled } from '~/utils/env.server';
+import { UserinfoToken } from '~/utils/raoidc-utils.server';
+import { getPathById } from '~/utils/route-utils';
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
```

Comment:
For all routes in the authenticated space, like this one, we need to add a `await raoidcService.handleSessionValidation(request, session);` in the `loader(..)`

Reply:
added